packages/fxa-settings/src/components/Ready/en.ftl

```diff
@@ -1,6 +1,8 @@
 ## Ready component
 
 reset-password-complete-header = Your password has been reset
+ready-complete-set-up-instruction = Complete set-up by entering the new password on your other { -brand-firefox } devices.
```

Comment:
Not blocking, as there are already 7 strings using "set-up", which I find very weird.

There are 9 strings using `setup`, which is what I would expect. Could we have content reviewing those strings, and updating them for consistency?

Reply:
Actually, there is a conversation happening in the other PR, pointing out that `set-up` is British spelling (introduced by Canadian developer?)
https://github.com/mozilla/fxa/pull/14877/

Maybe change this to `setup` before landing?

Reply:
Ah yes, and I'm the one who had pointed it out on the other ticket too! I'll definitely change it here. Thanks @flodolo!